builtin/credential/aws/backend.go

```diff
@@ -34,8 +34,14 @@ type backend struct {
 	// Lock to make changes to any of the backend's configuration endpoints.
 	configMutex sync.RWMutex
 
+	// Guards the initialize function
```

Comment:
I think you can pull these two things out as we should only ever call Initialize once per load